compiler/plugins/target/AMD-AIE/aie/test/nd_dma_distribute_AIE2.mlir

```diff
@@ -103,10 +103,10 @@
 
 module @ndDMAObjFifoAIE2 {
  aie.device(xcve2302) {
-    %tile10 = aie.tile(1, 0)
-    %tile11 = aie.tile(1, 1)
-    %tile22 = aie.tile(2, 2)
-    %tile23 = aie.tile(2, 3)
+    %tile10 = aie.tile(2, 0)
```

Comment:
There is no FileCheck in the run command for this test (not sure if accidentally missing or just a disabled test?)

Reply:
good catch